src/core/interfaces/ISablierLockup.sol

```diff
@@ -110,13 +113,24 @@ interface ISablierLockup is ISablierLockupBase {
     /// @notice Retrieves the segments used to compose the dynamic distribution function.
     /// @dev Reverts if `streamId` references a null stream or a non Lockup Dynamic stream.
     /// @param streamId The stream ID for the query.
+    /// @return segments See the documentation in {DataTypes}.
```

Comment:
Any reason why you decided to add `@return` tag here but not in above functions such as `getCliffTime`?

Reply:
to point out the reader to `DataTypes` file